src/meson.build

```diff
@@ -48,4 +48,18 @@ libglyphy = library('glyphy', glyphy_sources + glyphy_headers + glyphy_shader_so
   version: '0.0.0',
   install: true)
 
+libglyphy_dep = declare_dependency(
+  link_with: libglyphy,
+  include_directories: [confinc],
+  sources: glyphy_sources + glyphy_headers + glyphy_shader_sources)
```

Comment:
Why are you adding the original source files from libglyphy here? That will make the other project which links to libglyphy, both link to it and recompile all its sources a second time and include those as primary sources.

Reply:
Probably just because I copy pasta'd from a module also doing things wrong.